src/kernels/gpu_reference_kernel/naive_conv.cpp

```diff
@@ -28,6 +28,7 @@
 #include <hip/hip_runtime.h>
 #include <hip/hip_bfloat16.h>
 #endif
+#include <cfloat> // has DBL_EPSILON
```

Comment:
Can't include `<cfloat>` I believe because `hipRTC` does not support `std` headers.